### ISSUE TITLE
[ENHANCEMENT] Config - Use API to get config

### ISSUE THREAD

**Issue by damienderoistecso** [CSOIreland/PxStat#228]
**Describe the solution you'd like**
Use new API to read config rather than directly from file

**Comment by damienderoistecso** [comment]
@CHanamy57 

New api is "PxStat.Security.Configuration_API.Read"

This return two objects. One for global config, another for server config. In our "app.configuration.ajax.getFiles" function replace the two "api.ajax.config" functions for getting global and server configs with one api.ajax.jsonrpc.request where the call back populates our server and global config objects. The rest of the code should work as is from there

**Comment by damienderoistecso** [comment]
@neilfromcork Can you create a private RESTful version of "PxStat.Security.Configuration_API.Read" so we can read it from the client and merge with our client config

**Comment by neilfromcork** [comment]
RESTful version available for test - https://dev-ws.cso.ie/private/api.restful/PxStat.Security.Configuration_API.Read

**Comment by CHanamy57** [comment]
Configuration entity now displays global & server config from API rather than from File.

**Comment by damienderoistecso** [comment]
@CHanamy57 Following a benchmarking review, we have decided to revert to always reading config directly from files, both on server and client. This means we will have to revert any changes to the client side config entity to before we implemented the change to use the API to get the global and server config. Also the Reload config feature is now redundant and can be removed after we revert the changes.
